qds_sdk/cluster.py

```diff
@@ -274,6 +274,12 @@ def _parse_create_update(cls, args, action):
                                     help="location for ssh key to use to" +
                                          " login to the instance")
 
+        security_group.add_argument("--persistent-security-groups",
+                                    dest="persistent_security_groups",
+                                    help="a security group to associate with each" +
+                                         " node of the cluster. Typically used" +
+                                         " to provide access to external hosts")
+
```

Comment:
The input name is groups, but we support only one group at present. It's
been on my mind as well - maybe I'll change the name to
persistent_security_group itself.

On Tue, Mar 17, 2015 at 4:23 AM, Rohit Agarwal <notifications@github.com>
wrote:

> In qds_sdk/cluster.py
> <https://github.com/qubole/qds-sdk-py/pull/90#discussion_r26535565>:
>
> > @@ -274,6 +274,12 @@ def _parse_create_update(cls, args, action):
> >                                      help="location for ssh key to use to" +
> >                                           " login to the instance")
> >
> > +        security_group.add_argument("--persistent-security-groups",
> > +                                    dest="persistent_security_groups",
> > +                                    help="a security group to associate with each" +
> > +                                         " node of the cluster. Typically used" +
> > +                                         " to provide access to external hosts")
> > +
>
> The help says a security group... but the argument name is ...groups.
> Which one is correct? If the plural one is correct - let's also provide
> example usage in the help message.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qubole/qds-sdk-py/pull/90/files#r26535565>.
>